### DISCUSSION TITLE
如何在posts内容显示时，隐藏掉目录以外的widgets

### DISCUSSION THREAD

**Discussion by Dcx199302** [ppoffice/hexo-theme-icarus#1002] [Q&A]
就是如何实现官方的这个效果
![image](https://user-images.githubusercontent.com/61185183/147730417-6fcf6315-73f6-4d20-8cc4-a3524dc69bc5.png)
目前使用的是 4.0.4

我的文件目录结构如下
![image](https://user-images.githubusercontent.com/61185183/147731219-4a8f1051-08ce-4ec5-b928-d6dbd4d42b81.png)

**Comment by Dcx199302** [comment]
我在 scaffolds 的 page - draft 的头部配置成功了。

**Reply by dongeast** [reply]
您好，请问能详细说一下如何配置吗，我也遇到了这个困难并且和您的目录一样

**Reply by Dcx199302** [reply]
尝试看看楼下 @realzza 的说法吧， 我只是在scaffolds/draft.md   文件的最顶部做了局部的配置。应该不是最佳方案，我的github仓库有我写的配置，你可以clone下来。不过那并没什么特别的

**Reply by ppoffice** [reply]
https://github.com/ppoffice/hexo-theme-icarus/tree/site
https://github.com/ppoffice/hexo-theme-icarus/blob/site/_config.post.yml

**Comment by realzza** [comment]
和你的文件结构无关。
在`config.icarus.yml`中，将不需要的widget配置删掉即可。
